src/Generator.Converter/Main.cs

```diff
@@ -59,48 +59,22 @@ public int GetHashCode(XNode a)
             }
         }
     }
-    
+
     class EntryPoint
     {
+        static Options CLIOptions;
```

Comment:
Yes. However, the command line arguments are one of the few things that should perhaps be statically accessible to the program - it doesn't change, and it's the same across the entire program no matter the state.

Thoughts?